nuxeo-data-grid/nuxeo-data-grid.html

```diff
@@ -219,41 +219,69 @@
 
         static get properties() {
           return {
+            /**
+             * If enabled, it allows to select multiple documents and apply an action on them,
+             * like moving them to trash, to favorites or to a collection.
+             */
             multiSelection: {
               type: Boolean,
               value: true,
             },
-
+            /**
+             * If enabled, it displays the list of quickfilters defined on the associated
+             * page provider at the top of the search results.
+             */
             displayQuickFilters: {
               type: Boolean,
               value: false,
             },
-
+            /**
+             * Specify here a subset of quick filters in case you don't want
+             * all of them to be displayed on the search results.
```

Comment:
Instead of "in case you don't want all" it might be better "in case you want to specify the ones to".
Just because, afaik, you need to bind the PP quickfilters here so that they appear all.